packages/compass-import-export/src/modules/export.ts

```diff
@@ -546,17 +543,6 @@ export const runExport = ({
       type: ExportActionTypes.RunExportSuccess,
       aborted,
     });
-
-    // Don't emit when the data service is disconnected or not the same.
-    if (dataService === getState().dataService.dataService) {
-      dispatch(
-        globalAppRegistryEmit(
-          'export-finished',
```

Comment:
This event was used in tests but nowhere else and tests could just wait for the dispatched action instead, so I removed it